bf2fc6cc711aee1a0c2a/kas-fault-tests
# Add Openshift layer chaos tests

## Description
[MGDSTRM-4189](https://issues.redhat.com/browse/MGDSTRM-4189)

Addition of Kraken openshift-layer test to the fault testing pipeline.

As part of this issue, it was noticed that the Kraken tests were quite limited in their configuration, so the tests were updated to resolve this.

> **_IMPORTANT:_** this PR should be merged until the pipeline is in a working state (https://github.com/bf2fc6cc711aee1a0c2a/kas-fault-tests/pull/13) at which point this PR may need to be rebased. 

## Verification
- Run the pipeline using the `openshift` level tests parameter on the pipeline.
- On the target cluster, check that the pods in the `openshift-apiserver` have been deleted and new pods created.
- Ensure that the pipeline completes without error.
